pepper-apis/gae-egress-proxy/squid.conf

```diff
@@ -1,6 +1,8 @@
 # Access control definitions.
+acl dest_ports port 443                     # Regular SSL port
```

Comment:
Clarification on the port numbering- this is the regular HTTP SSL port (squid does support other protocols)